src/elements/content-sidebar/__tests__/ActivitySidebar.test.js

```diff
@@ -763,6 +764,26 @@ describe('elements/content-sidebar/ActivitySidebar', () => {
         });
     });
 
+    describe('handleAnnotationEdit()', () => {
+        test('should call updateAnnotation API', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+            instance.fetchFeedItems = jest.fn();
+
+            wrapper.instance().handleAnnotationEdit({
+                id: '123',
+                text: 'hello',
+                permissions: {
```

Comment:
Should we have a case where the permissions are not valid?

Reply:
I think we don't need to have that case in ActivitySidebar - it looks like we already have testing for invalid permissions that is within `/box/box-ui-elements/src/api/__tests__/Annotations.test.js`.